engine/classes/Elgg/Notifications/NotificationsService.php

```diff
@@ -32,18 +36,21 @@ class NotificationsService {
 	/**
 	 * Constructor
 	 *
-	 * @param Queue              $queue   Queue
-	 * @param PluginHooksService $hooks   Plugin hook service
-	 * @param \ElggSession       $session Session service
+	 * @param Queue              $queue       Queue
+	 * @param PluginHooksService $hooks       Plugin hook service
+	 * @param EventsService      $elgg_events Events service
+	 * @param \ElggSession       $session     Session service
 	 */
 	public function __construct(
 			Queue $queue,
 			PluginHooksService $hooks,
+			EventsService $elgg_events,
```

Comment:
why not add the new service at the end??

Reply:
just keeping hooks/events together... no specific reason